Bottom: 845d5ea04d64d48d669f488aae604f831577dfca
Top:    33e9cf93ebaeb1803b9f10dfd0bba6b69ba4ce0c
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 15:04:09 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index 412c9c8..302460f 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -981,6 +981,7 @@ static int parse_mapent(const char *ent, char *g_options, char **options, char *
 	}
 
 	if (!validate_location(logopt, loc)) {
+		error(LOGOPT_ANY, "revalidate_location failed");
 		free(myoptions);
 		free(loc);
 		return 0;
@@ -1014,6 +1015,7 @@ static int parse_mapent(const char *ent, char *g_options, char **options, char *
 		}
 
 		if (!validate_location(logopt, ent_chunk)) {
+			error(LOGOPT_ANY, "revalidate_location failed");
 			free(ent_chunk);
 			free(myoptions);
 			free(loc);
@@ -1442,6 +1444,7 @@ int parse_mount(struct autofs_point *ap, const char *name,
 			l = parse_mapent(p, options, &myoptions, &loc, ap->logopt);
 			if (!(strstr(myoptions, "fstype=autofs") &&
 			     strstr(myoptions, "hosts")) || !l) {
+				error(LOGOPT_ANY, "I think I'm a hosts map? l %d", l);
 				cache_delete_offset_list(mc, name);
 				cache_multi_unlock(me);
 				cache_unlock(mc);
